### ISSUE TITLE
Integrate accessibility into existing teams

### ISSUE THREAD

**Issue by mgifford** [CivicActions/accessibility#42]
All of our teams should have an accessibility champion & an understanding of their role on accessibility
- Design
- Front-end engineering
- Back-end engineering
- DevOps
- Customer Experience / Service Design 

### Resources
- USA https://accessibility.digital.gov/
    - [Content](https://accessibility.digital.gov/content-design/getting-started/) 
    - [Front End](https://accessibility.digital.gov/front-end/getting-started/)
    - [Product](https://accessibility.digital.gov/product/getting-started/) 
    - [UX](https://accessibility.digital.gov/ux/getting-started/) 
    - [Visual Design](https://accessibility.digital.gov/visual-design/getting-started/) 
- UK https://accessibility-manual.dwp.gov.uk/guidance-for-your-job-role
    - [Business Analyst](https://accessibility-manual.dwp.gov.uk/guidance-for-your-job-role/business-analyst)
    - [Content Designer](https://accessibility-manual.dwp.gov.uk/guidance-for-your-job-role/content-designer)
    - [Delivery Manager](https://accessibility-manual.dwp.gov.uk/guidance-for-your-job-role/delivery-manager)
    - [Digital Performance Analyst](https://accessibility-manual.dwp.gov.uk/guidance-for-your-job-role/digital-performance-analyst)
    - [Interaction Designer](https://accessibility-manual.dwp.gov.uk/guidance-for-your-job-role/interaction-designer)
    - [Product Manager](https://accessibility-manual.dwp.gov.uk/guidance-for-your-job-role/product-manager)
    - [QA Tester](https://accessibility-manual.dwp.gov.uk/guidance-for-your-job-role/qa-tester)
    - [Software Engineer or Frontend Developer](https://accessibility-manual.dwp.gov.uk/guidance-for-your-job-role/software-engineer-or-frontend-developer)
    - [User Researcher](https://accessibility-manual.dwp.gov.uk/guidance-for-your-job-role/user-researcher)

- W3C 
    - ARRM Project - [Accessibility Roles and Responsibilities Mapping](https://www.w3.org/WAI/EO/wiki/ARRM_Project_-_Accessibility_Roles_and_Responsibilities_Mapping)
    - [Accessibility Responsibility Breakdown](https://www.w3.org/community/wai-engage/wiki/Accessibility_Responsibility_Breakdown)

**Comment by mgifford** [comment]
Aggregated list

Back-end Engineer
Business Analyst
Content
Content Designer
Customer Experience / Service Design
Delivery Manager
DevOps
Digital Performance Analyst
Front End Engineer
Interaction Designer
Product Manager
QA Tester
User Researcher
UX
Visual Design

**Comment by mgifford** [comment]
Added information to 
https://github.com/CivicActions/accessibility/blob/main/_playbook/roles.md
